src/utils/queryObject-functions.ts

```diff
@@ -0,0 +1,38 @@
+//ignores queryFromURL and allowSearch.
+import { QueryObject } from '../types/search.types';
+
+export const queryObjectIsEquals = (a: QueryObject, b: QueryObject): boolean => {
```

Comment:
Får vi til å binde denne til queryObject slik "queryObject.equals(queryObject: QueryObject)"?
Sleit med at "this" kunne være undefined.